bundles.md

```diff
@@ -0,0 +1,131 @@
+# Attestation Bundle
```

Comment:
Link to this page from README.md. Also, should this go within `spec/`?

Reply:
You're suggesting it be moved to the spec folder?

Reply:
Done